site/app/templates/autograding/TAResults.twig

```diff
@@ -82,6 +82,11 @@ $( document ).ready(function() {
                             {% endif %}
                         {% endfor %}
                         {% for file in uploaded_pdfs %}
+                            {% if file.encoded_name in annotated_file_names %}
+                                {% if can_download %}
+                                    <button class = 'btn btn-default key_to_click' onclick='downloadStudentAnnotations("{{ student_pdf_download_url }}?filename={{ file.name }}&path={{ file.path|url_encode }}", "{{file.path|url_encode}}", "annotated_pdfs")' aria-label="Download the file"> Download
+                                        <i class="fas fa-download" title="Download the file"></i></button>
+                                {% endif %}
```

Comment:
```suggestion
                                {% endif %}
                            {% endif %}
```

Reply:
Also, the download button this adds breaks formatting, and it's not clear what each button would do without clicking them and looking at the result.

![Screenshot 2020-07-11 16 32 42](https://user-images.githubusercontent.com/1845314/87233322-ed4ac100-c3ce-11ea-82e1-c3ad3f83d300.png)

Reply:
The above if statement has be rewritten, as described below. This no longer applies